drivers/serial/uart_stm32.c

```diff
@@ -590,6 +590,10 @@ static int uart_stm32_err_check(const struct device *dev)
 		err |= UART_ERROR_FRAMING;
 	}
 
+	if (LL_USART_IsActiveFlag_LBD(UartInstance)) {
```

Comment:
It seems LDB detection is not available on some F0 variants.
This is then causing build issues on such targets.
To make it work on all SoCs, please put this code under following condition
```
#if !defined(CONFIG_SOC_SERIES_STM32F0X) || defined(USART_LIN_SUPPORT)
```